LICENSE

```diff
@@ -208,3 +208,6 @@
 This file is in the public domain.  For more information see:
 
 activemq-commons/src/main/java/org/apache/activemq/utils/Base64.java
+
+This product bundles HdrHistogram, which is available under a
+"2-clause BSD" license. For details, see https://github.com/HdrHistogram/HdrHistogram/blob/master/LICENSE.txt.
```

Comment:
This needs changed for a couple reasons.

This is almost certainly the wrong LICENCE file. This is for the source release. You prsumably are depending on a binary and so, if it is distributed, this would go in the binary archives LICENCE file, at _artemis-distribution/src/main/resources/licenses/bin/LICENSE_.

The licence itself needs to be included in the file, or in another file within the repo and referenced to (since the URL contents could change, e.g to a different licence, or just go away, e.g branch change without using the rename tool). Youll see lots of examples of the latter in the other file.

Reply:
I've re-used the same approach of the Apache Pulsar folks here: let me check why/what was the motivation (spoken with @gtully few months ago about it)

Reply:
> You prsumably are depending on a binary and so, if it is distributed, this would go in the binary archives LICENSE file, at artemis-distribution/src/main/resources/licenses/bin/LICENSE.

I see that now; yes good catch!